helm/templates/worker/deployment.yaml

```diff
@@ -17,6 +17,12 @@ spec:
     spec:
       serviceAccountName: controller-manager
       containers:
-        - name: wardle-server
+        - name: worker
           image: {{ .Values.worker.image.repository }}:{{ .Values.worker.image.tag }}
           imagePullPolicy: {{ .Values.worker.image.pullPolicy }}
+          volumeMounts:
+            - mountPath: /tmp
+              name: tmp-volume
+      volumes:
+        - name: tmp-volume
+          emptyDir: {}
```

Comment:
I didn't want to put this in memory as it also holds the trivy cache.
As a future improvement, we could expose the cache as a persistent volume so that the cache is persisted.

Reply:
Yes, that's totally fine with me. I think we don't need to have `/tmp` to be an emptyDir, this is already read-write inside of a Pod. Should we instead create a dedicated path for these operations? Like `/var/run/worker/`?